lib/tether/tether.go

```diff
@@ -679,6 +681,10 @@ func (t *tether) launch(session *SessionConfig) error {
 		case <-session.ClearToLaunch:
 			log.Infof("Received the clear signal to launch %s", session.ID)
 		}
+		// reset RunBlock to unblock process start next time
+		session.RunBlock = false
+		prefix := extraconfig.CalculateKeys(t.config, fmt.Sprintf("%s.%s", session.extraconfigKey, session.ID), "")[0]
+		extraconfig.EncodeWithPrefix(t.sink, session, prefix)
```

Comment:
do we need to call this? We are already calling EncodeWithPrefix in a defer statement above